src/pages/calendar/components/Section.tsx

```diff
@@ -82,16 +88,51 @@ export function SectionInfo({
                 Asynchronous
               </Badge>
             )}
+            {!isASYNC && !isBLENDED && instructionalMethod == 'online' && (
+              <Badge colorScheme="cyan" mx="1">
+                Synchronous
+              </Badge>
+            )}
+            {isBLENDED && (
```

Comment:
I'd say we could probably make these custom components that take in a boolean (like `isBLENDED`, `isASYNC`, etc.) and then conditionally render from that. Since this code is repeated so much it would clean it up.

Reply:
We can likely move this to the the backend entirely but I think for now it's fine for now.